recipe/meta.yaml

```diff
@@ -1,7 +1,6 @@
 {% set name = "pysoundfile" %}
 {% set pypi_name = "SoundFile" %}
-{% set version = "0.10.2" %}
-{% set sha256 = "637f6218c867b8cae80f6989634a0813b416b3e6132480d056e6e5a89a921571" %}
+{% set version = "0.10.3.post1" %}
```

Comment:
This doesn't seem to be a released version. Why not do 0.10.3 with a patch?

Reply:
It's the current version on PyPI and Debian has this one as well: https://packages.debian.org/sid/python3-soundfile (0.10.3+post1-1)

Reply:
OK, maybe this should go onto the rc label then?  I don't usually use any post/alpha/betas  on the main label?

Reply:
I don't understand? Why not put a post release on the main channel? Isn't it the equivalent of incrementing the build number in the pypi world? If Debian puts it in the main channel, I think we should, too.
It's the most recent version and was released over a year ago. Time for cf to catch up! :)

Reply:
Post-releases seem to be [for minor, non-code changes](https://www.python.org/dev/peps/pep-0440/#post-releases) so itr should be fine to either pull from the upstream `0.10.3` or rename post releases to the latest main release. 

I think it makes sense in general for conda-forge to have its own lifetimes for packages, and not be tied to other package managers.  

So just to be clear, I think it is fine to pull the post release, but the post number can either be ignored in the version field or become the build number.

Reply:
conda officially supports `.postX` version strings and the convenience is that we don't need to change the PyPI url etc. and autotickbot etc. should continue to work fine: https://github.com/conda/conda/blob/7132ac7a713a6d61d5da317d98cea961af748d75/conda/models/version.py#L65-L66

Reply:
Ok fair.